keyboards/bt66tech/bt66tech60/readme.md

```diff
@@ -0,0 +1,23 @@
+# bt66tech60 
+
+A Blue Pill STM32F103C8T6-based 60% ANSI board.
+
+Keyboard Maintainer: [Andrew Kannan](https://github.com/bt66)  
```

Comment:
This file should match the template <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md>

Also is the name here, `Andrew Kannan`, correct?